docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
   web:
     container_name: whist-server
-    image: whist-team/whist-server:latest
+    image: teamwhist/whist-server:latest
```

Comment:
Why change team name? I think it is required to work with you.

Reply:
The old name didn't work at all for me